docs/core/logging.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Logging
+description: Core utility
+---
+
+Logger provides an opinionated logger with output structured as JSON.
+
+**Key features**
+
+* Capture key fields from Lambda context, cold start and structures logging output as JSON
+* Log Lambda event when instructed (disabled by default)
+    - Enable explicitly via annotation param
```

Comment:
Is the additional level intentional? If yes can we move it up since it only refer to the line before, something like that:
* Log Lambda event when instructed, disabled by default, can be enabled explicitly via annotation param